AsyncDisplayKit/Details/Transactions/_ASAsyncTransaction.mm

```diff
@@ -13,6 +13,8 @@
 #import <list>
 #import <map>
 
+NSUInteger const ASDefaultTransactionPriority = 128;
```

Comment:
Yeah, NSUInteger didn't feel right from the beginning. I'll change it to NSInteger with default to zero. Makes much more sense.